airflow/models/dag.py

```diff
@@ -630,30 +567,37 @@ def get_run_dates(self, start_date, end_date=None):
         :type start_date: datetime
         :param end_date: the end date of the interval, defaults to timezone.utcnow()
         :type end_date: datetime
+        :param align: whether the first run should be delayed to "align" with
+            the schedule, or can happen immediately at start_date (default: True
+            for top-level dags, False for subdags)
+        :type align: bool
```

Comment:
This is _forced_ to false fo subdags -- do we need to set the default to be `align = None` maybe?

Reply:
This can *never* be True for subdags, otherwise the scheduler would stuck. I’ll change the wording instead.